gvm/protocols/gmpv8.py

```diff
@@ -541,3 +546,219 @@ def modify_tag(
                 _xmlresources.add_element("type", resource_type)
 
         return self._send_xml_command(cmd)
+
+    def clone_ticket(self, ticket_id):
+        """Clone an existing ticket
+
+        Arguments:
+            result_id (uuid): Result the ticket applies to
+            assigned_to_user_id (uuid): User the ticket is assigned to
+            open_note (str): A note about opening the ticket
+            comment (str, optional): Comment for the ticket
+
+        Returns:
+            The response. See :py:meth:`send_command` for details.
+        """
+        if not ticket_id:
+            raise RequiredArgument("clone_ticket requires a ticket_id argument")
+
+        cmd = XmlCommand("create_ticket")
+
+        _copy = cmd.add_element("copy", ticket_id)
+
+        return self._send_xml_command(cmd)
+
+    def create_ticket(
+        self, *, result_id, assigned_to_user_id, open_note, comment=None
+    ):
+        """Create a new ticket
+
+        Arguments:
+            result_id (uuid): Result the ticket applies to
+            assigned_to_user_id (uuid): User the ticket is assigned to
+            open_note (str): A note about opening the ticket
+            comment (str, optional): Comment for the ticket
+
+        Returns:
+            The response. See :py:meth:`send_command` for details.
+        """
+        if not result_id:
+            raise RequiredArgument(
+                "create_ticket requires a result_id argument"
+            )
+
+        if not assigned_to_user_id:
+            raise RequiredArgument(
+                "create_ticket requires a assigned_to_user_id argument"
+            )
+
+        if not open_note:
+            raise RequiredArgument(
+                "create_ticket requires a open_note argument"
+            )
+
+        cmd = XmlCommand("create_ticket")
+
+        _result = cmd.add_element("result")
+        _result.set_attribute("id", result_id)
+
+        _assigned = cmd.add_element("assigned_to")
+        _user = _assigned.add_element("user")
+        _user.set_attribute("id", assigned_to_user_id)
+
+        _note = cmd.add_element("open_note", open_note)
+
+        if comment:
+            cmd.add_element("comment", comment)
+
+        return self._send_xml_command(cmd)
+
+    def delete_ticket(self, ticket_id, *, ultimate=False):
+        """Deletes an existing ticket
+
+        Arguments:
+            ticket_id (str) UUID of the ticket to be deleted.
```

Comment:
Is there a reason for documenting the type of `ticket_id` as `str`? `create_ticket` above documents `id`s as `uuid`, this should be consistent IMHO.

There are other occurences below.

Reply:
Just double checked with the gmpv7 part and we use str as type for uuids currently. Therefore I've adjusted the docstrings accordingly.